### ISSUE TITLE
ms3 logger overwrites default logger [BUG]

### ISSUE THREAD

**Issue by apmcleod** [johentsch/ms3#68]
**Describe the bug**
Importing ms3 sets the logging level and it cannot be changed back somehow?

**ms3 version**
1.2.8

**To Reproduce**
```
$ python
> import logging
> logging.basicConfig(level=logging.DEBUG)
> logging.debug("test")  # Logs correctly
```

New session:
```
$ python
> import logging
> import ms3
> logging.basicConfig(level=logging.DEBUG)
> logging.debug("test")  # Nothing is printed
```

**Comment by johentsch** [comment]
Thanks for reporting, @apmcleod. I'm on it.

**Comment by johentsch** [comment]
fixed in v1.2.10. Could you please confirm, @apmcleod ?

**Comment by apmcleod** [comment]
Yes, it's fixed!
